ps_featuredproducts.php

```diff
@@ -72,6 +72,8 @@ public function install()
             && $this->registerHook('deleteproduct')
             && $this->registerHook('categoryUpdate')
             && $this->registerHook('displayHome')
+            && $this->registerHook('displayOrderConfirmation2')
```

Comment:
Yes, we know, but we have two hooks on the same page and we need do differentiate them.
You can use the widget system by editing the template but we have to register default hooks on module install to make a default configuration.